R/twobit.R

```diff
@@ -71,9 +67,10 @@ setMethod("export", c("DNAStringSet", "TwoBitFile"),
             if (any(width(object) == 0L)) {
               stop("Empty strings are not yet supported")
             }
-            invisible(.TwoBits_export(mapply(.DNAString_to_twoBit, object,
-                                             seqnames),
-                                      twoBitPath(path(con))))
+            Rtwobitlib::twobit_write(
+                x = as.character(object),
+                filepath = twoBitPath(path(con))
+            )
```

Comment:
@hpages 
The write operation here is simplified and does not make use of any `masks(object)`. I'm not sure how to apply those to the operation.